src/CONST.ts

```diff
@@ -1,29 +1,29 @@
-import lodashGet from 'lodash/get';
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
why do we need this rule @blazejkustra ?

Reply:
There are some number keys in CONST:
```ts
HEIGHT_FOR_ROW_COUNT: {
            1: 60,
            2: 80,
            3: 100,
        },
```